### DISCUSSION TITLE
Seems to be missing Exchange documentation?

### DISCUSSION THREAD

**Discussion by wushu037** [rabbitmq/rabbitmq-website#1661] [General]
### Is your feature request related to a problem? Please describe.

Hi, where should I look for documentation on Exchange?
I didn't find pure Exchange documentation on the Docs page
Nor did I find what I'm looking for in golang's ExchangeDeclare comments

For example, I'm now wondering "when using the default Exchange (set to `""`), the routingKey should be set to the queue name so that messages are routed to the corresponding queue", which is what I'm guessing, since it matches the code in the example. So I'm trying to find reliable documentation

Thanks!

### Describe the solution you'd like

I want documentation that specifically describes Exchange

**Comment by michaelklishin** [comment]
There is no separate guide on exchanges. There are [several tutorials](https://rabbitmq.com/getstarted.html), including an [introductory one](https://rabbitmq.com/tutorials/amqp-concepts.html). They cover all key exchange types, including the default exchange.

There is also a guide on [Publishers](https://rabbitmq.com/publishers.html), which mentions exchanges 44 times according to a quick search.

**Reply by wushu037** [reply]
hi, @michaelklishin 
I have read several tutorials and Publisher's guide before asking the question
Several tutorials have examples of the default Exchange
but neither of them explicitly mentions the paragraph I guessed

My go code is simple, it's actually the examples from several tutorials, and I just ran a copy of those examples

**Reply by michaelklishin** [reply]
The default exchange is automatically void to every queue in the same virtual host, with the routing key equal to queue name.

This provides a certain illusion that you are publishing "directly to the queue". Some clients have API methods that claim that they "publish to a queue", even thought that is not actually true for AMQP 0-9-1 clients.

**Comment by michaelklishin** [comment]
It would be much easier to suggest something specific if you had posted a snippet of your Go code.

**Comment by lukebakken** [comment]
https://www.rabbitmq.com/tutorials/amqp-concepts.html#exchanges

https://www.rabbitmq.com/tutorials/amqp-concepts.html#exchange-default

**Reply by wushu037** [reply]
> https://www.rabbitmq.com/tutorials/amqp-concepts.html#exchanges
> 
> https://www.rabbitmq.com/tutorials/amqp-concepts.html#exchange-default

Thanks, this is exactly what I was looking for, great!

**Reply by lukebakken** [reply]
👍 for your information, I found that document with the following google search:

https://www.google.com/search?q=site%3Arabbitmq.com+default+exchange

**Reply by wushu037** [reply]
Haha, thank you, it seems I should master some more search methods
